hack/validation/kubelet.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+set -euo pipefail
+
+# Kubelet Validation (placeholder)
```

Comment:
Add this to the follow ups? Or is this meant as part of the "and tests" for kubelet part?

Reply:
This is for special cases of CEL validation, which is already on the list